staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -737,8 +737,8 @@ func (g *Cloud) SetInformers(informerFactory informers.SharedInformerFactory) {
 		UpdateFunc: func(prev, obj interface{}) {
 			prevNode := prev.(*v1.Node)
 			newNode := obj.(*v1.Node)
-			if newNode.Labels[v1.LabelZoneFailureDomain] ==
-				prevNode.Labels[v1.LabelZoneFailureDomain] {
+			if newNode.Labels[v1.LabelTopologyZone] ==
```

Comment:
This changed from beta->GA which is probably safe but might be good to keep this in a separate PR.